src/Carbon/Doctrine/CarbonTzImmutableType.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Carbon\Doctrine;
+
+use Carbon\CarbonImmutable;
+use Carbon\CarbonInterface;
+use Doctrine\DBAL\Platforms\AbstractPlatform;
+use Doctrine\DBAL\Types\DateTimeTzImmutableType;
+
+class CarbonTzImmutableType extends DateTimeTzImmutableType implements CarbonDoctrineType
+{
+    use CarbonTypeConverter;
+
+    public function getName(): string
+    {
+        return 'carbontz_immutable';
+    }
+
+    public function convertToDatabaseValue(mixed $value, AbstractPlatform $platform): ?string
+    {
+        return $this->convertCarbonToDatabaseValue($value, $platform);
```

Comment:
This method does not exist, did you mean to alias the one imported from the trait?

```php
use CarbonTypeConverter {
    convertToDatabaseValue as convertCarbonToDatabaseValue
}

Reply:
done